framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/database/driver/AbstractExternalDriver.java

```diff
@@ -31,6 +31,11 @@ public String getTargetDockerTag(URL dockerUrl) {
 
     @Override
     public String getCekitImageBuildCommand() {
-        return "make build " + getName();
+        return "make build " + getName() + " BUILD_ENGINE=docker";
+    }
+
+    @Override
+    public String getJdbcDriverUrl() {
```

Comment:
What about making the response Optional? That would reflect the fact that the driver URL doesn't have to be specified.

Reply:
Changed